requirements/base.txt

```diff
@@ -42,3 +42,8 @@ fake-factory==0.5.7
 whitenoise==2.0.6
 django-compressor==2.0
 django-libsass==0.6
+
+# Async Tasks
+# -------------------------------------
+celery[redis]==3.1.23
+django-celery-email==1.1.5
```

Comment:
let's not use celery worker here, to keep the stack simple. 

I would prefer to use python async backend from http://bameda.github.io/djmail/#_backends